src/test/java/com/faforever/client/game/TeamCardControllerTest.java

```diff
@@ -0,0 +1,75 @@
+package com.faforever.client.game;
+
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.player.Player;
+import com.faforever.client.replay.Replay.PlayerStats;
+import com.faforever.client.test.AbstractPlainJavaFxTest;
+import com.faforever.client.theme.UiService;
+import com.faforever.client.util.Rating;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableMap;
+import javafx.scene.control.Label;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TeamCardControllerTest extends AbstractPlainJavaFxTest {
+  private TeamCardController instance;
+  @Mock
+  private Player player1;
+  private ArrayList<Player> list;
+  @Mock
+  private I18n i18n;
+  @Mock
+  private UiService uiService;
+  @Mock
+  private PlayerCardTooltipController playerCardTooltipController;
+  @Mock
+  private RatingChangedLabelController ratingChangedLabelController;
+
+  private ObservableMap<String, List<PlayerStats>> teams = FXCollections.observableHashMap();
```

Comment:
Initialize this in `setUp` for consistency.
And you trapped in your own trap; this doesn't need to be an `ObservableMap` but should just be `Map`. Then you can use an ordinary `HashMap` or any other you like